.github/workflows/stalebot.yml

```diff
@@ -50,3 +53,93 @@ jobs:
                   enable-statistics: true
                   ascending: true
                   operations-per-run: 120
+            - name: Remove stale PR approvals (over 30 days old)
+              uses: actions/github-script@v7
+              with:
+                  script: |
+                      const thirtyDaysAgo = new Date();
+                      thirtyDaysAgo.setDate(thirtyDaysAgo.getDate() - 30);
+                      console.log(`Checking for approvals before: ${thirtyDaysAgo.toISOString()}`);
+
+                      const prs = await github.paginate(github.rest.pulls.list, {
+                        owner: context.repo.owner,
+                        repo: context.repo.repo,
+                        state: 'open',
+                        per_page: 100
+                      });
+
+                      for (const pr of prs) {
+                        const reviews = await github.paginate(github.rest.pulls.listReviews, {
+                          owner: context.repo.owner,
+                          repo: context.repo.repo,
+                          pull_number: pr.number
+                        });
+                        
+                        // Find the latest approval date for this PR
+                        let latestApproval = null;
+                        const approvalReviews = [];
+                        
+                        for (const review of reviews) {
+                          if (review.state === 'APPROVED') {
+                            const reviewDate = new Date(review.submitted_at);
+                            approvalReviews.push(review);
+                            if (!latestApproval || reviewDate > latestApproval) {
+                              latestApproval = reviewDate;
+                            }
+                          }
+                        }
+                        
+                        // Process only if we found any approvals
+                        if (latestApproval) {
+                          console.log(`PR #${pr.number} latest approval on ${latestApproval.toISOString()}`);
+                          
+                          if (latestApproval < thirtyDaysAgo) {
```

Comment:
I think we can safely combine these two if's via `if ( latestApproval && latestApproval < thirtyDaysAgo )` while achieving the same effect in terms of the logging of the PR number? This will save us one level of nested if's. This is more of a nit-pick though.